src/Avalonia.Visuals/Point.cs

```diff
@@ -267,5 +267,24 @@ public Point WithY(double y)
         {
             return new Point(_x, y);
         }
+
+        /// <summary>
+        /// Deconstructor for decomposition Point
+        /// </summary>
+        /// <param name="x">The X position.</param>
+        /// <param name="y">The Y position.</param>
+        public void Deconstruct(out double x, out double y)
+        {
+            x = this._x;
+            y = this._y;
+        }
+
+        /// <summary>
+        /// Gets a value indicating that point coordinates are zero
```

Comment:
I'd prefer "Gets a value indicating whether the X and Y coordinates are zero." ("that" is confusing). Similarly for the other `IsDefault` methods.